csv-data-management/extract_data_from_csv.py

```diff
@@ -10,28 +10,29 @@
 from uuid import uuid4
 import requests
 import pandas as pd
```

Comment:
do you still need pandas?

Reply:
> do you still need pandas?
For the Action, Convert JSON to CSV, JSON Field. There is no polars function which can linearize Nested JSON Fields having a record path and Meta variables, It would simply give an error. So I had to use pandas.json_normalize function at that action only. If it's given that there will be no nested data within the JSON provided, I can change it to polars.